component/component.go

```diff
@@ -20,7 +20,7 @@ import (
 	"go.opentelemetry.io/collector/config"
 )
 
-// Component is either a receiver, exporter, processor or an extension.
+// Component is either a receiver, exporter, processor, or extension.
```

Comment:
Inconsistent with package comment, I think you need to keep `an`

Reply:
Addressed this, thanks